tests/test_events.py

```diff
@@ -216,3 +229,36 @@ def test_missing_answer_id(self):
         self.example_event['event']['answers'] = {}
         SubmissionRecorder().send(self.example_event)
         self.assert_unsuccessful_event_parsing()
+
+    @pytest.mark.usefixtures("example_event")
+    def test_open(self):
+        """
+        Events should be recorded only when the problem is open
+        """
+        event = self.example_event
+        event_before = combine_dicts(event, {'test_data': 'before'})
+        event_during = combine_dicts(event, {'test_data': 'during'})
+        event_after = combine_dicts(event, {'test_data': 'after'})
+
+        problem_id = UsageKey.from_string(event['event']['problem_id'])
+        course_id = CourseLocator.from_string(event['context']['course_id'])
+
+        block_status = RapidResponseBlockStatus.objects.get(
+            usage_key=problem_id,
+            course_key=course_id,
+        )
```

Comment:
You're fetching the `RapidResponseBlockStatus` that you created in `setUp` here, right? If so, I think (1) it makes more sense to just set a property on the object (something like self.example_event_block_status`) and access it in this test case, and (2) you could just get `problem_id` and `course_id` from that object instead of using `UsageKey.from_string`/`CourseLocator.from_string`